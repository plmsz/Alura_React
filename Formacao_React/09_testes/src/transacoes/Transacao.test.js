import React from "react";
import Transacao from "./Transacao";
import { render } from "@testing-library/react";

xdescribe("Componente de transação do extrato", () => {
	it("O snapshot do componente deve permanecer sempre o mesmo", () => {
		const { container } = render(
			<Transacao data="01/03/2022" tipo="deposito" valor="100" />
		);
		expect(container.firstChild).toMatchSnapshot();
	});
});

