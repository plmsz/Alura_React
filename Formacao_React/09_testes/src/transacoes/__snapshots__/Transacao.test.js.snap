// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente de transação do extrato O snapshot do componente deve permanecer sempre o mesmo 1`] = `
<div
  class="transacao-container"
>
  <p>
    01/03/2022
  </p>
  <p>
    deposito
  </p>
  <p>
    R$ 
    100
  </p>
</div>
`;
